main.go

```diff
@@ -435,6 +442,12 @@ func getHubManager(
 				&v1.Secret{}: {
 					Field: fields.SelectorFromSet(fields.Set{"metadata.name": secretsync.SecretName}),
 				},
+				&extensionsv1.CustomResourceDefinition{}: {
```

Comment:
Sorry I didn't catch this before but this needs to be in the `getManager` method and not the `getHubManager` method. See my git diff I previously posted.